### ISSUE TITLE
Ability to place and use ammunition in the inventory

### ISSUE THREAD

**Issue by SampeyFS** [TheGameCreators/GameGuruRepo#4918]
No weapons or ammunition can be placed in the inventory, especially ammunition that cannot be placed in the inventory and does not give the possibility to select collectables

**Comment by SampeyFS** [comment]
Weapons and ammunition cannot be placed in the shop and are not visible in the inventory

**Comment by LeeBamberTGC** [comment]
Closing this as reported correctly as BUG elsewhere. You should be able to have weapons in your inventory.

**Comment by SampeyFS** [comment]
no, the ammo doesn't work ((ammo doesn't let you add collectibles)) @LeeBamberTGC

**Comment by LeeBamberTGC** [comment]
You should be able to have weapons in your inventory.
